test/test_helper.rb

```diff
@@ -0,0 +1,13 @@
+ENV["RAILS_ENV"] ||= "test"
```

Comment:
You can get rid of the `test/` directory, if you are only interested in RSpec tests

Reply:
Is it not a directory that I might need down the line, even if I'm only interested in RSpec tests currently?

Reply:
We don't really use MiniTest at Smart so you may be better off spending your efforts on learning RSpec; you won't regret it